templates/default/kafka_mirror_maker_initd.erb

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+<%# coding: UTF-8%>
+# Auto-generated by Chef, do not edit!
+### BEGIN INIT INFO
+# Provides: kafka-mirror-maker
+# Short-Description: Kafka mirror maker server
+# Default-Start: 3 4 5
+# Default-Stop: 0 1 2 6
+# Required-Start:
+# Required-Stop:
+# Should-Start:
+# Should-Stop:
+### END INIT INFO
+
+USER=<%= node["kafka"]["user"] %>
+SLEEP_TIME=5
+CURRENT_WAIT=0
+TIMEOUT=<%= node["kafka"]["shutdown_timeout"] %>
+KAFKA_MAIN=kafka.tools.MirrorMaker
+
+# Setup JMX port environment variable, if set
+<% if !node["kafka"]["mirror_maker"]["env_vars"]["JMX_PORT"].nil? and !node["kafka"]["mirror_maker"]["env_vars"]["JMX_PORT"].empty? -%>
```

Comment:
I think this is the only place where you use this so I don't think the other env vars would get honored